gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -309,6 +309,8 @@
   /**
    * Converter configuration properties.
    */
+  public static final String CONVERTER_AVRO_INCLUDE_SCHEMA_CREATION_TIME = "converter.avro.include.schema.creation.time";
```

Comment:
Do we need to make it configurable ?

Reply:
this is only a key name for the config value. Will rename it to avoid confuse.